pkg/crc/machine/bundle/metadata.go

```diff
@@ -238,17 +238,20 @@ func GetCustomBundleName(bundleFilename string) string {
 	return fmt.Sprintf("%s_%d%s", baseName, time.Now().Unix(), bundleExtension)
 }
 
-func GetBundleNameFromURI(bundleURI string) string {
+func GetBundleNameFromURI(bundleURI string) (string, error) {
 	// the URI is expected to have been validated by validation.ValidateBundlePath first
```

Comment:
This comment can be removed as it did not prevent this bug from occurring, and this is now no longer required.

Reply:
Done!